http-parser.js

```diff
@@ -133,7 +135,7 @@ HTTPParser.prototype.execute = function (chunk, start, length) {
   length = this.offset - start;
   if (headerState[this.state]) {
     this.headerSize += length;
-    if (this.headerSize > HTTPParser.maxHeaderSize) {
+    if (this.headerSize > (this.maxHeaderSize || HTTPParser.maxHeaderSize)) {
```

Comment:
Would be good to use `??` here (modern syntax).

Reply:
@oleavr updated